src/test/scripts/functions/paramserv/mnist_lenet_paramserv_minimum_version.dml

```diff
@@ -295,9 +295,8 @@ predict = function(matrix[double] X, int C, int Hin, int Win,
 
   # Compute predictions over mini-batches
   probs = matrix(0, rows=N, cols=K)
-  batch_size = 64
   iters = ceil(N / batch_size)
-  for(i in 1:iters) {
+  parfor(i in 1:iters, check=0) {
```

Comment:
That is great - on a scenario of mnist60K, 2 epochs per worker, ASP, EPOCH, batchsize=64, 80 workers, this change alone change the end-to-end runtime from 61s to 36s.